packages/logrotate/build

```diff
@@ -10,6 +10,12 @@ make
 make install
 
 
+# Create the module file
+module="$PKG_PATH/dcos.target.wants_slave/logrotate.json"
```

Comment:
I'm guessing you want this to be `$PKG_PATH/etc/mesos-slave-modules/` so it comes out where you want